### PR TITLE
Fixes SSL issue and additionally adds in some code for eRMSD calculation

### PR DESCRIPTION
**Pull request by hlasimpk**
For UNIX systems I'm checking if the SSL_CERT_FILE environmental variable has been set and if not I am setting it manually. I'm doing this in each of the command line scripts but if you can think of a better way to do it let me know. 

The eRMSD code is still a work in progress as I need to decide on what my defaults are, but it should be an improvement over the current code as it takes the size of the search model into account when determining the RMSD.

### CODE REVIEW

**Comment by fsimkovic** [comment]
I can't comment on the files itself, but for the SSL error move the check to `simbad.__init__.py`, which is the best place to put it. This fixes it when the SIMBAD environment is loaded, regardless of which module is initiated.

File: simbad/rotsearch/phaser_search.py [hunk 1]
```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
...
+
+        if id < 0 or id > 100:
+            msg = "Sequence id must be a number between 0 and 100"
+            raise RuntimeError(msg)
+
+        if nres < 0:
+            msg = "The number of residues must be greater than 0"
+            raise RuntimeError(msg)
+
+        return np.around((1.54967686e+00 + 1.69034597e-03 * nres + -2.57055256e-02 * id + -5.92896517e-07 * nres ** 2
```
Comment (fsimkovic) [inline simbad/rotsearch/phaser_search.py]: Is this the equation they use for the eRMSD?

File: simbad/rotsearch/phaser_search.py [hunk 2]
```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
...
+                          + -1.68255006e-05 * nres * id + 2.03689221e-04 * id ** 2 + 1.01170096e-10 * nres ** 3
+                          + 2.45170962e-09 * nres ** 2 * id + 5.10824965e-08 * nres * id ** 2
+                          + -7.01029519e-07 * id ** 3), decimals=3)
```
Comment (fsimkovic) [inline simbad/rotsearch/phaser_search.py]: Wouldn't round the value to a specific number of decimals, users could do that or we can do it when printing the output, but you're not allowing anyone to access the full value

**Comment by hlasimpk** [comment]
Thanks, I thought this might be better suited in a different place! I'll move it now

File: simbad/rotsearch/phaser_search.py [hunk 3]
```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
         summarize_result(self.search_results,
                          csv_file=csv_file, columns=columns)
 
+    @staticmethod
+    def calculate_ermsd(nres, id):
+        """Calculate the estimated rmsd based on the number of residues and sequence id"""
+
+        if id < 0 or id > 100:
```
Comment (fsimkovic) [inline simbad/rotsearch/phaser_search.py]: Non-essential but could re-write this to `0 > id < 100`. I think this works and is generally easier to read plus quicker if done many times (which it is not here but just in general)

File: simbad/rotsearch/phaser_search.py [hunk 4]
```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
...
+            msg = "Sequence id must be a number between 0 and 100"
```
Comment (fsimkovic) [inline simbad/rotsearch/phaser_search.py]: Maybe add the message directly into the exception call

Comment (fsimkovic) [reply simbad/rotsearch/phaser_search.py]: Same goes for the call a little below

File: simbad/rotsearch/phaser_search.py (hunk 1 above)
Comment (hlasimpk) [reply simbad/rotsearch/phaser_search.py]: No, they just give the graph of values from which I plotted the points and fitted a curve to.

File: simbad/rotsearch/amore_search.py [hunk 5]
```diff
@@ -169,11 +169,16 @@ def run(self, models_dir, nproc=2, shres=3.0, pklim=0.5, npic=50, rotastep=1.0,
             name = os.path.basename(dat_model.replace(".dat", ""))
             pdb_struct = simbad.util.pdb_util.PdbStructure()
             pdb_struct.from_file(dat_model)
-            solvent_content = sol_calc.calculate_from_struct(pdb_struct)
-            if solvent_content < min_solvent_content:
-                msg = "Skipping %s: solvent content is predicted to be less than %.2f"
-                logger.debug(msg, name, min_solvent_content)
-                continue
+            try:
```
Comment (fsimkovic) [inline simbad/rotsearch/amore_search.py]: Fair enough for now but would dig into the reason for some of those failing. Worth pinging Andrey an email as well to check with him what's going on.

**Pull request merged**